extension/searching.js

```diff
@@ -12,6 +12,49 @@ this.searching = (function() {
     return searchUrl.href;
   };
 
+  exports.ddgEntitySearch = async function(query) {
+    const response = await fetch(
+      `https://api.duckduckgo.com/?q=${query}&format=json&pretty=1&skip_disambig=1`
+    );
+    const ddgData = await response.json();
+    if (!ddgData.AbstractText) {
+      // the response from DDG was null, and there was no matching Instant Answer result
+      return {};
```

Comment:
Should this return null?

Reply:
Ah, yep! Sorry. Did a dumb thing to make the linter happy.